### PR TITLE
add etcd kv service test

### PR DESCRIPTION
**Pull request by AdoHe**

### CODE REVIEW

**Comment by AdoHe** [comment]
@xiang90 @hongchaodeng @skyao @sijie could someone help fix testng issue? I am not familiar with testng.

**Comment by sijie** [comment]
I am not familiar with testng too. I am wondering why we didn't use junit?

**Comment by AdoHe** [comment]
@skyao seems you add testng, could you help with this?

**Comment by xiang90** [comment]
i think @StupidHod is trying testng. Can we make sure we only use one testing framework?

**Comment by AdoHe** [comment]
@xiang90 I am ok about testng.

File: scripts/run_etcd.sh [hunk 1]
```diff
@@ -1,12 +1,28 @@
 #!/bin/bash
 
-mkdir -p external
```
Comment (hongchaodeng) [inline scripts/run_etcd.sh]: @AdoHe What do you think about separating this into another PR? We don't need to block this.

Comment (AdoHe) [inline scripts/run_etcd.sh]: that's ok.

**Pull request merged**